lib/teiserver/libs/test_lib.ex

```diff
@@ -92,6 +92,12 @@ defmodule Teiserver.TeiserverTestLib do
 
   @spec auth_setup(nil | Map.t()) :: %{socket: port(), user: Map.t(), pid: pid()}
   def auth_setup(user \\ nil) do
+    # Remember to call Teiserver.Client.disconnect(user.id) at the end
```

Comment:
I think this is just a workaround to issue where teiserver instance is shared across all tests. I do not think it is wise to share a teiserver instance and with every tests figureout how to correctly clean it up. We should instead ensure teiserver boots and shuts down on each test that requires connections to teiserver.